test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -491,7 +491,7 @@ func testWebhookAdmission(t *testing.T, watchCache bool) {
 		// force enable all resources so we can check storage.
 		"--runtime-config=api/all=true",
 		// enable feature-gates that protect resources to check their storage, too.
-		"--feature-gates=EphemeralContainers=true",
+		// e.g. "--feature-gates=EphemeralContainers=true",
```

Comment:
Why keep this as a comment vs deleting it? Curious, really

Reply:
I remember having a lot of trouble tracking down this problem so I wanted to leave some documentation for the next person, but I don't feel strongly about it.

Reply:
SGTM